Backend/GettingStartedNode/hello_world/task.md

```diff
@@ -13,7 +13,7 @@ which contains all the essential information about the project, and a source cod
 ### index.js
 This file is where all the magic of web development happens.
 
-First, we create an HTTP server using the `createServer` method and specify a handler function for all incoming requests.
+First, we create an HTTP server using the `createServer` method and specify a callback handler function for all incoming requests.
```

Comment:
Let's make `callback` a hyperlink to https://developer.mozilla.org/en-US/docs/Glossary/Callback_function just to make it even more clear for the learners